File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rsp/xLink/api/XLinkImpl.java
package org.ngbp.jsonrpc4jtestharness.rsp.xLink.api;

import org.ngbp.jsonrpc4jtestharness.rsp.drm.model.NotifyParams;
import org.ngbp.jsonrpc4jtestharness.rsp.mapper.JsonRpcResponse;

public class XLinkImpl implements XLink {
    @Override
    public JsonRpcResponse<Object> xLinkResolutionNotification() {
        return null;
    }

    @Override
    public JsonRpcResponse<NotifyParams> xLinkResolved() {
        return null;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/xLink/XLinkImpl.kt
package org.ngbp.jsonrpc4jtestharness.rpc.xLink

import org.ngbp.jsonrpc4jtestharness.rpc.RpcResponse
import org.ngbp.jsonrpc4jtestharness.rpc.drm.model.NotifyParams

class XLinkImpl : IXLink {
    override fun xLinkResolutionNotification(): RpcResponse {
        return RpcResponse()
    }

    override fun xLinkResolved(): NotifyParams {
        return NotifyParams()
    }
}